packages/cli/src/cmds/validator/voluntaryExit.ts

```diff
@@ -33,7 +33,7 @@ If no `pubkeys` are provided, it will exit all validators that have been importe
 
   examples: [
     {
-      command: "validator voluntary-exit --pubkeys 0xF00",
+      command: "validator voluntary-exit --network goerli --pubkeys 0xF00",
```

Comment:
Aligned with [validator-import](https://chainsafe.github.io/lodestar/reference/cli/#validator-import) command and makes it more apparent that this command is network specific